### ISSUE TITLE
Recurring crash possibly due to powercell/fluxduct interaction?

### ISSUE THREAD

**Issue by popps73x** [FTBTeam/FTB-Continuum#88]
modpack Version 1.1.0 (the same issue was happening in the previous version also however)

https://www.dropbox.com/s/8u4iv3ek98rxpqa/crash-2018-05-25_08.41.42-server.txt?dl=0

to me this appears to be the Energy cells? (storage item from rftools power) not playing well with the Leadstone Flux Ducts from Thermal.  this is the 4th crash report I have and they all seem to read the same both in this updated version and the previous one.  

The setup I am using.  2 water wheels (IE) with 3 leadstone fluxducts moving power to 4 energy cells (set to input)  power is extracted with IE LV connectors and wire from the multiblock (set to output).  the IE wires then attach to small sets of Leadstone Fluxducts to distribute to "groups" of machines.

**Comment by LezChap** [comment]
Had a similar crash...any time I tried to place a low-tier powercell next to 2 coal generators from Vulpes it'd crash to desktop.  Crash occurred whenever I placed a powercell next to an existing one, but only if the coal generators were there to fill it.  If I removed the coal generators, placed the cell, then replaced the coal generators, I'd never produce the crash.

crashlog: https://gist.github.com/LezChap/17c9325744903b2fc927d879f64e9277
